governance.md

```diff
@@ -338,10 +338,11 @@ Council that the changes be ratified and the pull request merged
 (accepting the proposed changes) or proposes that the pull request be
 closed without merging (rejecting the proposed changes).  The Member
 should state the final commit hash in the pull request being proposed
-for acceptance or rejection and briefly summarize the pull request. A
-minimum of 80% of the Steering Council must vote and at least 2/3 of
-the votes must be positive to carry out the proposed action (fractions
-of a vote rounded up to the nearest integer). Since the BDFL holds
-ultimate authority in The Project, the BDFL has authority to act alone
-in accepting or rejecting changes or overriding Steering Council
-decisions.
+for acceptance or rejection and briefly summarize the pull request. All votes
+are limited in time to 4 weeks after the vote is initiated. At the end of 4
+weeks, the proposal passes if 2/3 of the votes are in favor (fractions of a
+vote rounded up to the nearest integer); otherwise the proposal is rejected and
+the PR is closed. Prior to the four-week limit, if 80% of the Steering Council
+has voted in favor, the proposal passes. Since the BDFL holds ultimate
```

Comment:
```suggestion
has voted  and 2/3 of the votes are in favor, the proposal passes. Since the BDFL holds ultimate
```

Reply:
The reasoning for the 80% vote was a sort of short-circuit of the one-month interval, i.e., the vote passes by acclaim. Do you think it should instead be: short-circuit the one-month limit if 80% of the SC votes with 2/3 of the votes being "for" the motion?

Reply:
> Do you think it should instead be: short-circuit the one-month limit if 80% of the SC votes with 2/3 of the votes being "for" the motion?

Correct. Because, to me, the reasoning for the one-month interval is to short-cirtcuit the *lack* of 80% voting requirement that's currently in place :)

So if 80% of folks vote before a month has passed, it's a equivalent to the current bylaws with my amendment, whereas as written without it, the threshold for 80% positive is higher, and more decisions will take a full month before "expiring" to the 2/3rd of all those who voted.